src/modules/client/pages/ni/planning/CustomerPlanning.vue

```diff
@@ -176,6 +176,11 @@ export default {
     openCreationModal (vEvent) {
       const { dayIndex, person } = vEvent;
       const selectedDay = this.days[dayIndex];
+
+      if (moment(selectedDay).toDate().toISOString() > person.stoppedAt) {
```

Comment:
Tu peux utiliser `DateHelper.dateDiff` ici, non ?

Reply:
heu c'est pas trop la même fonction, datediff calcule le temps entre deux dates et toDate transforme un objet moment en date. Faudrait refacto this.days et ça sort un peu du périmètre du ticket non ?